AdminPanelService/AdminPanel.API/Controllers/CarModelController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using AdminPanel.BLL.CQS.CatalogueService.Commands.CarModelCommands.AddCarModel;
+using AdminPanel.BLL.CQS.CatalogueService.Commands.CarModelCommands.DeleteCarModel;
+using AdminPanel.BLL.CQS.CatalogueService.Commands.CarModelCommands.UpdateCarModel;
+using AdminPanel.BLL.CQS.CatalogueService.Queries.CarModelQueries.GetCarModelById;
+using AdminPanel.BLL.CQS.CatalogueService.Queries.CarModelQueries.GetCarModelsInRange;
+
+namespace AdminPanel.API.Controllers;
+
+[ApiController]
+[Route("api/model")]
+public class CarModelController(ISender sender) : ControllerBase
+{
```

Comment:
May be it will be better to create an additional field here to use it instead of `sender` variable?
something like `private readonly ISender _sender = sender`

Reply:
Primary constructor does exactly the same thing, so I believe there's no need in additional field

Reply:
Like, it's the same private readonly field

Reply:
It's a question more about naming